app/assets/stylesheets/devise-edit.scss

```diff
@@ -22,7 +22,7 @@ h2 {
 }
 
 input {
-  background: linear-gradient(black, grey);
+  background-color: grey;
   color: white;
```

Comment:
You don't have to change this now. But try get into the habit of using hex codes for your colours.

Example:
Use #808080 for grey
And #fff for white
etc.